packages/swc-helpers/src/_object_spread_props.js

```diff
@@ -0,0 +1,30 @@
+function ownKeys(object, enumerableOnly) {
+  var keys = Object.keys(object);
+  if (Object.getOwnPropertySymbols) {
```

Comment:
https://caniuse.com/mdn-javascript_builtins_object_getownpropertysymbols

Hmm, it's not supported by ie11

Reply:
It's for getting object symbol keys, and IE11 doesn't have symbol to begin with